bin/lib/localized-urls.json

```diff
@@ -1,151 +1 @@
"https:/resources.scratch.mit.edu/www/cards/es/Scratch2Cards.pdf","cards.catchCardsLink":"https:/resources.scratch.mit.edu/www/cards/es/catchCards.pdf","cards.danceCardsLink":"https:/resources.scratch.mit.edu/www/cards/es/danceCards.pdf","cards.fashionCardsLink":"https:/resources.scratch.mit.edu/www/cards/es/fashionCards.pdf","cards.flyCardsLink":"https:/resources.scratch.mit.edu/www/cards/es/flyCards.pdf","cards.hide-seekCardsLink":"https:/resources.scratch.mit.edu/www/cards/es/hide-seekCards.pdf","cards.musicCardsLink":"https:/resources.scratch.mit.edu/www/cards/es/musicCards.pdf","cards.nameCardsLink":"https:/resources.scratch.mit.edu/www/cards/es/nameCards.pdf","cards.petCardsLink":"https:/resources.scratch.mit.edu/www/cards/es/petCards.pdf","cards.pongCardsLink":"https:/resources.scratch.mit.edu/www/cards/es/pongCards.pdf","cards.raceCardsLink":"https:/resources.scratch.mit.edu/www/cards/es/raceCards.pdf","cards.storyCardsLink":"https:/resources.scratch.mit.edu/www/cards/es/storyCards.pdf"},"fr":{"cards.Scratch2CardsLink":"https:/resources.scratch.mit.edu/www/cards/fr/Scratch2Cards.pdf"},"hr":{"cards.Scratch2CardsLink":"https:/resources.scratch.mit.edu/www/cards/hr/Scratch2Cards.pdf"},"it":{"cards.Scratch2CardsLink":"https:/resources.scratch.mit.edu/www/cards/it/Scratch2Cards.pdf"},"ja":{"cards.Scratch2CardsLink":"https:/resources.scratch.mit.edu/www/cards/ja/Scratch2Cards.pdf"},"ja-hr":{"cards.Scratch2CardsLink":"https:/resources.scratch.mit.edu/www/cards/ja-hr/Scratch2Cards.pdf"},"ko":{"cards.Scratch2CardsLink":"https:/resources.scratch.mit.edu/www/cards/ko/Scratch2Cards.pdf"},"nl":{"cards.Scratch2CardsLink":"https:/resources.scratch.mit.edu/www/cards/nl/Scratch2Cards.pdf"},"pt-br":{"cards.Scratch2CardsLink":"https:/resources.scratch.mit.edu/www/cards/pt-br/Scratch2Cards.pdf"},"sl":{"cards.Scratch2CardsLink":"https:/resources.scratch.mit.edu/www/cards/sl/Scratch2Cards.pdf"},"sv":{"cards.Scratch2CardsLink":"https:/resources.scratch.mit.edu/www/cards/sv/Scratch2Cards.pdf"},"zh-tw":{"cards.catchCardsLink":"https:/resources.scratch.mit.edu/www/cards/zh-tw/catchCards.pdf","cards.danceCardsLink":"https:/resources.scratch.mit.edu/www/cards/zh-tw/danceCards.pdf","cards.fashionCardsLink":"https:/resources.scratch.mit.edu/www/cards/zh-tw/fashionCards.pdf","cards.flyCardsLink":"https:/resources.scratch.mit.edu/www/cards/zh-tw/flyCards.pdf","cards.hide-seekCardsLink":"https:/resources.scratch.mit.edu/www/cards/zh-tw/hide-seekCards.pdf","cards.musicCardsLink":"https:/resources.scratch.mit.edu/www/cards/zh-tw/musicCards.pdf","cards.nameCardsLink":"https:/resources.scratch.mit.edu/www/cards/zh-tw/nameCards.pdf","cards.petCardsLink":"https:/resources.scratch.mit.edu/www/cards/zh-tw/petCards.pdf","cards.pongCardsLink":"https:/resources.scratch.mit.edu/www/cards/zh-tw/pongCards.pdf","cards.raceCardsLink":"https:/resources.scratch.mit.edu/www/cards/zh-tw/raceCards.pdf","cards.storyCardsLink":"https:/resources.scratch.mit.edu/www/cards/zh-tw/storyCards.pdf"}}
```

Comment:
Is this something that might need to get updated by hand? Should it be pretty printed? (Easy way in node is to just get the object and run `JSON.stringify(obj, null, '    ')` with the last argument being the tab character.